app/javascript/components/CardStack.js

```diff
@@ -3,23 +3,32 @@ import PropTypes from "prop-types"
 import {CardFace, CardBack, CardSpot} from './Card'
 import DropTarget from "./DropTarget";
 import { postEvent } from "../modules/utils"
-import {addEvent, updateCard, watch, unWatch} from '../state/CardState';
+import {addEvent, updateCard} from '../state/CardState';
+import { stack } from '../state/atoms'
+import {useRecoilValue} from 'recoil';
 
 const CardStack = (props) => {
+  const stackCards = useRecoilValue(stack(props.locationId + '-' + props.stack))
```

Comment:
Commenting out this line removes the warning... it also results in nothing being rendered